test/node/common/index.js

```diff
@@ -0,0 +1,122 @@
+// https://github.com/nodejs/node/blob/c975384264dc553de62398be814d0c66fc1fc1fb/test/common/index.js
+
```

Comment:
Is the `assert.js` based on a Node file? If so the commit comment like this addd to that is helpful.